docs/get-started.md

```diff
@@ -46,6 +48,53 @@ NOTE: Make sure to check out the appropriate version. We recommend that you chec
     kubectl get deployments --namespace=nginx-example
     ```
 
+### (Optional) Expose Minio outside your cluster
+
+When you run commands to get logs or describe a backup, the Ark server generates a pre-signed URL to download the requested items. To access these URLs from outside the cluster -- that is, from your Ark client -- you need to make Minio available outside the cluster. You can:
+
+- Change the Minio Service type from `ClusterIP` to `NodePort`.
+- Set up Ingress for your cluster, keeping Minio Service type `ClusterIP`.
+
+In Ark 0.10, you can also specify the value of a new `publicUrl` field for the pre-signed URL in your backup storage config.
+
+#### Expose Minio with Service of type NodePort
+
+The Minio deployment by default specifies a Service of type `ClusterIP`. You can change this to `NodePort` to easily expose a cluster service externally if you can reach the node from your Ark client.
+
+You must also get the Minio URL, which you can then specify as the value of either the `s3Url` or the new `publicUrl` field in your backup storage config.
```

Comment:
IMHO it's confusing to say you can specify either but not explain why or what (if anything) would change, this is why I wanted to just be updating `publicUrl`.  I'd either explain that in this case you can change either because Ark inside the cluster should be able to access the nodeport URL, or change the text to specify changing one of them (personally I'd choose `publicUrl`).